NewForce-at-Mountwest/bangazon-workforce-management-the-lemony-slaps
# Finished Create Employee

Description:
Given the user is viewing the list of employees
When the user clicks the Create New link
Then a form for be displayed on which the following information can be entered

First name
Last name
Is the employee a supervisor
Select a department from a drop down

If you want to see the ticket for yourself then click the link or copy and paste the link below:
https://trello.com/c/q2LeOkRl/2-2-hr-should-be-able-to-add-an-employee


Steps to test:

Go ahead and pull down these changes to your branch.
Once you've done that, go ahead and execute and run the app.
The next thing to do is to change the url by adding /employees/Create in order to see the form for creating a new employee.
The form should load up, and you can then fill out the first and last name of a new employee, and what department they will work in.
There is also a little check box right above the submit button that a user can select if the new employee is a supervisor.
Once the form is completely filled out, you can then hit the submit button.
You should then be taken to a list of all of the employees in the Bangazon work force.

Cheers.
